common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -609,6 +609,159 @@ func (s *ExecutionManagerSuite) TestUpdateWorkflowExecutionStateCloseStatus() {
 	}
 }
 
+// TestUpdateWorkflowExecutionTTL test
+func (s *ExecutionManagerSuite) TestUpdateWorkflowExecutionTTL() {
+	//TODO: Update the test when the TTLbuffer becomes configurable.
```

Comment:
Can we make sure the test only runs when the database is Cassandra?

Reply:
example https://github.com/uber/cadence/blob/master/common/persistence/persistence-tests/matchingPersistenceTest.go#L558